### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Azure Sign In_ Keyboard: User has to press again spacebar even after selecting the “Service Principal” radio button using arrow key to enable the “Browse” and “New” controls.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3324]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** MAS2.1.1

**Repro Steps:** 
1) Open IntelliJ IDEA application
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application.
7)Tab till “Azure Sign In” button and press enter.
8)Select “Service Principle” radio button using arrow key and verify whether the “Browse” and “New” controls are enabled or not.

**Actual Result:**
User has to press again spacebar even after selecting the “Service Principal” radio button using arrow key to enable the “Browse” and “New” controls.

**Expected Result:**
Whenever user  selects “Service Principal” radio button using arrow keys then only “Browse” and “New” controls should get enabled.

**User Impact:**
Keyboard users don’t think that they need to press again Spacebar to activate those controls. They think that those will be enabled automatically when we select radio button.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
[5.Service.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451191/5.Service.pptx)

**Comment by Flanker32** [comment]
This is a common behavior of IntelliJ, users need to press `Space` to ensure their selection and then ide will show the enabled items. Here is a screen shot:
![3324](https://user-images.githubusercontent.com/12445236/64147470-ef5cfe80-ce52-11e9-8543-5215ec2f6973.gif)

**Comment by Flanker32** [comment]
- Importance of experience: Medium,users will meet this situation when sign in.
- Quality of experience: Little, this may be a little confused, but sometimes users may just want to select another item instead of the following one, press `space` to confirm will help them in this case.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Accessibility56 Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked with [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us review it, thanks!

**Comment by Accessibility56** [comment]
**Logged an External bug here:**
https://youtrack.jetbrains.com/issue/IDEA-222111

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749781
